src/context.rs

```diff
@@ -162,6 +162,47 @@ impl Context {
         self.sessions
     }
 
+    /// Get current capability information about the TPM.
+    pub fn get_capabilities(
+        &mut self,
+        capability: TPM2_CAP,
+        property: u32,
+        property_count: u32,
+    ) -> Result<(TPMS_CAPABILITY_DATA, bool)> {
+        let mut outcapabilitydata = null_mut();
+        let mut outmoredata: u8 = 0;
+
+        let ret = unsafe {
+            Esys_GetCapability(
+                self.mut_context(),
+                self.sessions.0,
+                self.sessions.1,
+                self.sessions.2,
+                capability,
+                property,
+                property_count,
+                &mut outmoredata,
+                &mut outcapabilitydata,
+            )
+        };
+        let moredata = if outmoredata == 0 {
+            false
+        } else if outmoredata == 1 {
+            true
+        } else {
+            panic!("Invalid outmoredata value");
```

Comment:
It's probably better to return an error here instead of panicking. Maybe creating a new variant in `WrapperErrorKind` - `WrongValueFromTpm` or something like that?

Reply:
Hah, that was my idea indeed. And then I left this from my first iteration.